src/queries.js

```diff
@@ -109,7 +109,7 @@ function queryAllByText(
   if (typeof container.matches === 'function' && container.matches(selector)) {
     baseArray = [container]
   }
-  return Array.from([...baseArray, ...container.querySelectorAll(selector)])
+  return Array.from([...baseArray, ...Array.from(container.querySelectorAll(selector))])
```

Comment:
We don't need the `return Array.from([` that can simply be: `return [`

Reply:
True!